webui/react/src/components/ProjectCard.module.scss

```diff
@@ -2,18 +2,14 @@
   display: grid;
   gap: 16px;
   grid-template:
-    'header  header' auto
+    'header header' auto
     'workspace workspace' 20px
     'footer footer' 20px / auto auto;
   height: 100%;
 
-  .headerContainer {
+  :global(.ant-typography-single-line) {
```

Comment:
still prefer to not use a selector that's coupled with the hew implementation. can we just wrap the `Title` in a div with a class? or I think this could be avoided by using `Column`s and `Row`s instead of CSS grid.

Reply:
✅